module/x/peggy/client/cli/tx.go

```diff
@@ -29,6 +29,8 @@ func GetTxCmd(storeKey string) *cobra.Command {
 	peggyTxCmd.AddCommand([]*cobra.Command{
 		CmdWithdrawToETH(),
```

Comment:
this CmdWtihdrawtoETH actually does the eth sending, I forgot we had it when I wrote my issue. It needs testing and a rename, probably good otherwise.